cmd/monaco/download/download_command.go

```diff
@@ -50,6 +50,8 @@ func GetDownloadCommand(fs afero.Fs, command Command) (cmd *cobra.Command) {
 			return preRunChecks(f)
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
```

Comment:
Probably makes no functional difference, so I'd merge this and address it after 2.0 release: 
In other commands we set this up in the PreRun function - would it make sense to silence this in the pre-run as well? Or should we ruther not silence other commands in the pre-run, but on run? 
See the usages of cmd.SilenceUsageCommand

Reply:
Tried to put as another command, but did not work. Another command doesn't use PreRunE, probably that's the reason.